content/en/docs/reference/setup-tools/_index.md

```diff
@@ -1,4 +1,4 @@
 ---
-title: Setup tools
+title: Set up tools
```

Comment:
```suggestion
title: Setup tools
```

Reply:
This suggestion is incorrect... 
This section describes "how to set up tools", not "What the setup tools are", so the abbreviated title should use the verb form "Set up tools", not the adjective "Setup tools".

But I'm removing this fix from the PR because fixing the page title would probably also require the folder to be renamed from "setup-tools" to "set-up-tools".